### PR TITLE
Feature/form styled

### PR DESCRIPTION
**Pull request by Killuina**

### CODE REVIEW

File: src/components/Form/FormStyled.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    align-items: center;
+    gap: 0.7rem;
+  }
+
+  label {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    padding: 0.5rem;
+  }
```
Comment (benjaminrae) [inline src/components/Form/FormStyled.tsx:25]: Enter!

**Comment by Killuina** [comment]
[x] Remove Form from App
[x] Change input label to username
[x] Add theme with global styles
[x] Add styles to form component

**Review by benjaminrae — APPROVED** [review]
@Killuina sigue con esto: Testea el componente formulario (responsabilidad de mostrar).

**Comment by Killuina** [comment]
[x] Add spacing

**Pull request merged**